spring-kafka/README.md

```diff
@@ -0,0 +1,357 @@
+# Spring Kafka
+
+## Goal
+The goal of this module is to be able to have a common code base to send and/or receive messages from a Kafka broker
+using Spring Kafka as the underlying library.
+
+This module also enforces that Google Protobuf is used as the message encoding.
+
+## Type Converter
```

Comment:
It is hard to understand why we're talking about Type Converters from the get go. This is a peculiarity that should be addressed when that context is relevant. 
I propose to start with the big "boxes" of the communication of Producer => Broker => Consumer and talk a bit about how message contract is specified (protobuf).

Reply:
Ok, I will start with a short introduction and talk about the converters in the end.